package.json

```diff
@@ -9,7 +9,7 @@
     "test:orig": "echo \"Error: no test specified\" && exit 1"
   },
   "engines": {
-    "node": "^10.0.0",
+    "node": ">=^10.0.0",
```

Comment:
I don't think the carat should be there. If you want to have an upper limit, you just do a ` <x.x.x` after it.

Reply:
Good call. I think other than the carat its the correct syntax ```For example, the comparator >=1.2.7 would match the versions 1.2.7, 1.2.8, 2.5.3, and 1.3.9, but not the versions 1.2.6 or 1.1.0.```

Reply:
We should match to nutella since that is the consumer and I am looking to enforce 12.x. So we should prob make this repo reflect that as well?

Reply:
I disagree. The version should reflect the minimum needed for the package to build and run. This package has no knowledge of Nutella and should not be bound by its requirements. Adding unnecessary restriction just adds complication. This worked fine in v8, too.

Reply:
Normally I'd say yeah, that should be the case, but if this is only used by our code base, we should make sure the node versions are close to the same or just remove the version all together knowing that nutella is going to reinforce the min node version. It would make sense that we just kill the min requirement all together; however this has been merged and I don't think it's worth any effort except to delete this repo :)

Reply:
I'll defer to the [documentation for package.json](https://docs.npmjs.com/files/package.json#engines) on this one.

> You can specify the version of node that your stuff works on

Additionally, we enforce the version in Nutella mostly as a carryover from when we used to check compiled code into the codebase. With the compiled code, minor variations in the Node version would produce slightly different code from each machine that resulted in significant merge conflicts. It's really no longer necessary to be strict about the versioning except perhaps during the actual builds where we _may_ want to have more control just for consistency.

(I'll copy this over to the main PR as well.)